app/Process.php

```diff
@@ -2,13 +2,36 @@
 
 namespace App;
 
+use Illuminate\Support\Carbon;
+
 class Process
 {
     protected $commands = [];
     protected $cwd = null;
     protected $asUser = null;
     protected $output = true;
 
+    /** @var resource $outputFile */
+    protected $outputFile;
+
+    public function __construct()
+    {
+        $filename = rtrim(sys_get_temp_dir() . '/fwd_output_' . Carbon::now()->format('Ymdhis'));
```

Comment:
I guess it is worth we use some `uniqid()` at the end to avoid names collisions.